src/sa_web/static/js/routes.js

```diff
@@ -138,17 +150,40 @@ var Shareabouts = Shareabouts || {};
 
     // Open view for first step in multi-step form
     newPlace: function() {
+      console.log("newPlace");
+
       this.appView.newPlace();
     },
 
     viewLandmark: function(id) {
+      console.log("view landmark");
+
       this.appView.viewLandmark(id, { zoom: this.loading });
     },
 
+    // decide whether to route to the place view or the landmark view
+    // REFACTOR
+    // This method replaces viewLandmark and viewPlace
+    viewPlaceOrLandmark: function(datasetSlug, id, responseId) {
+      console.log("viewPlaceOrLandmark");
+
+      // REFACTOR
+      // Using hard-coded values here to decide whether to route to viewPlace
+      if (datasetSlug == "report" || datasetSlug == "place") {
```

Comment:
Yeah, eventually this would look something like `if (datasetSlug is in this.placesDatasets) {`